roles/install-bridge-cni/tasks/main.yml

```
@@ -0,0 +1,40 @@
+- name: Create cni directory
+  file:
+    path: /opt/cni/bin
+    state: directory
+
+- name: Download and setup cni plugins
+  unarchive:
+    src: "{{ cni_plugins_url }}/{{ cni_plugins_tarball }}"
```

Referenced from the suggestion in `group_vars/all`
```suggestion
    src: "{{ cni_plugins_url }}/{{ cni_plugins_version }}/{{ cni_plugins_tarball }}"
```